src/library/coverartcache.h

```diff
@@ -26,11 +26,13 @@ class CoverArtCache : public QObject, public Singleton<CoverArtCache> {
     QPixmap requestCover(const CoverInfo& info,
                          const QObject* pRequestor,
                          const int desiredWidth,
+                         const double devicePixelRatio,
```

Comment:
Please finish the renaming in the header file: desiredWidth -> deviceIndependentWidth